packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -877,6 +916,9 @@ function ensureRootIsScheduled(root: FiberRoot, currentTime: number) {
 
   root.callbackPriority = newCallbackPriority;
   root.callbackNode = newCallbackNode;
+  if (enableFrameEndScheduling) {
+    root.frameAlignedNode = newFrameAlignedNode;
```

Comment:
This shouldn't be a separate field. There's only ever a single task scheduled per root, so we only need one field to store a task handle.

Reply:
Done